### PR TITLE
Add the ability to control service status

### PR DESCRIPTION
**Pull request by ron-benaim**
Add the ability to control service status

Fixes https://github.com/theforeman/puppet-dhcp/issues/153

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -37,6 +37,7 @@
   Hash[String, Hash] $hosts = {},
   Variant[Array[String], Optional[String]] $includes = undef,
   String $config_comment = 'dhcpd.conf',
+  Optional[String] $service_ensure = undef,
```
Comment (ekohl) [inline manifests/init.pp]: To remain compatible it should stay the same:
```suggestion
  String $service_ensure = 'running',
```

Comment (ron-benaim) [reply manifests/init.pp]: If you would like that puppet wont change the state of the service, what value will you set?
Using undef will not change the service status, or, we can set another "if" about whether puppet should manage the service or not.
WDYT?

Comment (ekohl) [reply manifests/init.pp]: Good point. How about:
```puppet
Enum['running', 'unmanaged', 'stopped'] $service_ensure = 'running',

$_service_ensure_real = $service_ensure ? {
  'unmanaged' => undef,
  default     => $service_ensure,
}
```

Comment (ron-benaim) [reply manifests/init.pp]: Looks great :)

Comment (ekohl) [reply manifests/init.pp]: Could you come up with the changes and idea also some tests?

File: manifests/init.pp [hunk 2]
```diff
@@ -154,8 +155,13 @@
...
 
+  $_service_ensure_real = $service_ensure ? {
+    'unmanaged' => undef,
+    default     => $service_ensure,
+  }
+  
   service { $servicename:
-    ensure => running,
+    ensure => $_service_ensure_real,
     enable => true,
```
Comment (ekohl) [inline manifests/init.pp]: You should determine this as well. Having `ensure => stopped` but `enable => true` is inconsistent.

**Comment by zjhuntin** [comment]
Closing for inactivity

**Pull request closed without merging**